server.js

```diff
@@ -0,0 +1,77 @@
+
+let bodyParser = require('body-parser');
+let express = require('express');
+
+let app = express();
+let port = 8080;
+let buzzWords = [];
+let justBuzzWords = [];
```

Comment:
This doesn't need to be fixed, but try and think of a way accomplish the same functionality of justBuzzwords without having to use this array.